core/arch/x86/decode_table.c

```diff
@@ -1458,9 +1458,9 @@ const instr_info_t * const op_instr[] =
 #define H15_dq TYPE_H, OPSZ_15_of_16
 #define Hqq TYPE_H, OPSZ_32
 #define Hx TYPE_H, OPSZ_16_vex32
-#define Hvq_q TYPE_H, OPSZ_half_16_vex32
+#define Hvd_dq TYPE_H, OPSZ_half_16_vex32
```

Comment:
Per ongoing offline discussion: IMHO having a 'vd' here is not appropriate, as it means 'double precision floats inside a vector of size 128 or 256 depending on vex' all by itself.  I don't think the 'v{ds}', 'p{ds}', or 's{ds}' specifiers should be used for fractional size types: it's too confusing b/c they're already talking about fractional components but implying the full size.  Here the underscore is showing the "N_of_M" fractional nature already, and we don't care much about the type (single or double float), so I would argue this should just be a plain size letter.

Plus, we should use 'x' instead of 'dq' since 'x' means '16_vex32'.

If we had a letter that meant 'half' (or one that meant '8_vex16') we could use it here.  Maybe add 'h', so `Hh_x`?

Reply:
It looks like we're not using 'x' today.  Maybe Intel added that one to their tables *after* I added AVX2?  Hmm, and I use it for `Fx`.

Reply:
My bad: we are using `Vx`, `Wx`, `Hx`, `Lx`.

Reply:
Per offline discussion. New half type now Hh_x, Wh_x. Added vmovddup to the fix.